src/Opdex.Platform.WebApi/Controllers/AuthController.cs

```diff
@@ -24,6 +33,50 @@ public AuthController(AuthConfiguration authConfiguration, IMediator mediator)
             _mediator = mediator ?? throw new ArgumentNullException(nameof(mediator));
         }
 
+        /// <summary>
+        /// Stratis Open Auth Protocol
+        /// </summary>
+        /// <remarks>Responds to a request from a Stratis Open Auth Signer.</remarks>
+        /// <param name="query">Tne Stratis Open Auth query string.</param>
+        /// <param name="body">The Stratis Open Auth body.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        /// <response code="200">Signature was validated successfully.</response>
+        /// <response code="400">The request is not valid.</response>
+        [HttpPost]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<IActionResult> StratisOpenAuthCallback([FromQuery] StratisOpenAuthCallbackQuery query,
+                                                                 [FromBody] StratisOpenAuthCallbackBody body, CancellationToken cancellationToken)
+        {
+            var expectedCallbackPath = System.IO.Path.Combine(_authConfiguration.StratisOpenAuthProtocol.CallbackBase, AuthPath);
+            var expectedId = new StratisId(expectedCallbackPath, query.Uid, query.Exp);
```

Comment:
OpdexConfiguration has the API URL callback base per network, but will need the https:// stripped out. Would be better to reuse that vs a new config.

That config also has `WalletTransactionCallback` set to `/transactions`.

Consider doing something similar like that here so the auth config points to `/auth` vs the const string in the controller should this logic ever move outside of the controller.

Reply:
Updated to reuse config and retrieve callback path from config